crates/oxc_parser/src/lexer/mod.rs

```diff
@@ -420,11 +420,10 @@ impl<'a> Lexer<'a> {
             }
             '^' => self.read_caret(),
             '#' => {
-                // https://tc39.es/proposal-hashbang/out.html
```

Comment:
it's part of the spec now https://tc39.es/ecma262/multipage/ecmascript-language-lexical-grammar.html#sec-hashbang